libs/platform/src/lib/components/list/list.component.ts

```diff
@@ -404,6 +404,16 @@ export class ListComponent extends CollectionBaseInput implements OnInit, AfterV
             item._noSeperator = this.noSeperator;
             this.stateChanges.next(item);
         });
+
+
+        const indicators = this.itemEl.nativeElement.querySelectorAll('fd-busy-indicator');
+        if (indicators) { 
```

Comment:
`querySelectorAll` always returns NodeList object, there is no sense to check if `indicators` is empty